src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $space-between-cards;
+  }
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    color: $color-main;
+  }
+
+  &__value {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__cash {
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-main;
+    line-height: 18px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    text-decoration: none;
+    color: #fff;
+    text-transform: uppercase;
+    text-align: center;
+    background-color: $blue-accent;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+  }
+
+  &__button:hover {
+    border-color: $blue-accent;
+    color: $blue-accent;
+    transition: 0.3s linear;
```

Comment:
add property for which you need to set transition
by default it will be set for all properties, which will affect in bad way on website performance